atriaapp/atriacalendar/models.py

```diff
@@ -105,8 +105,18 @@ def __str__(self):
         return self.label
 
 
+# Base class for organizations that use the Atria platform
+class AtriaOrganization(models.Model):
+    org_name = models.CharField(max_length=40)
+    date_joined = models.DateTimeField(default=timezone.now)
+    status = models.CharField(max_length=8)
+    description = models.CharField(max_length=4000)
```

Comment:
For Charfields above a certain size, it''s probably better to use `models.TextField`, as it better informs `ModelForm` widget resolution.

Reply:
Changed to TextField